src/controllers/routers/referees.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * OneSG API Server by TL Bootcamp#6 OneSG Team
+ * Copyright(c) 2020 TechLadies
+ * MIT Licensed
+ */
+
+'use strict';
+
+const { check, validationResult } = require('express-validator');
+
+const {
+  errors: { BadRequest, UnprocessableEntity },
+} = require('../../utils');
+
+const db = require('../../../config/db');
+
+/**
+ * Retrieve all references
+ * @param {Request} req
+ * @param {Response} res
+ */
+const getAll = async (req, res) => {
+  const result = await db('referees');
+  res.status(200).json(result);
+};
+
+/**
+ * Create new reference
+ * @param {Request} req
+ * @param {Response} res
+ */
+const create = async (req, res) => {
+  // Return if there are any validation errors
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res
+      .status(new BadRequest().error.statusCode)
+      .json(new BadRequest(errors));
+  }
+  const newReferee = req.body;
+  try {
+    // Check if reference already exists in db, and returns RefereeId
+    await db('referees')
+      .insert(newReferee)
+      .returning('RefereeId')
+      .then((id) => {
+        return res.json({ message: 'Reference successfully created', id });
+      });
+    return res.status(201);
```

Comment:
nice use of 201 :)

Reply:
we don't need the message btw, since with a 201 status, the client should already know that the referee was created successfully

Reply:
removed json message

Reply:
removed message and replace with RefereeId